src/RenderWebGL.js

```diff
@@ -842,11 +875,16 @@ class RenderWebGL extends EventEmitter {
             };
         }
 
+        // TODO: figure out why this is all in a try block?
```

Comment:
IIRC this try-catch dates back to cwillisf's initial implementation of the code. What error is it supposed to catch?